### ISSUE TITLE
Automated Text Classification (relevant labeled data and moded definitions)

### ISSUE THREAD

**Issue by RohitRathore1** [the-vision/jarvis-core#58]
In reference of issue no #57
## Uber Ludwig
Completing the plan I described `in issue #57` using Deep Learning generally requires writing advanced Python code. Fortunately, Uber has released a super valuable tool called [Ludwig]( https://eng.uber.com/introducing-ludwig/) that makes it possible to build and use predictive models with incredible ease. We will run Ludwig from within [Google Colaboratory](https://colab.research.google.com/) in order to use their free GPU runtime. Training Deep Learning models without using GPUs can be the difference between waiting a few minutes to waiting hours. 

## Automated Text Classification
In order to build predictive models, we need relevant labeled data and moded definitions. So, you will practice with a simple text classification model straight from the [Ludwig examples](https://uber.github.io/ludwig/examples/). We are going to use a labeled dataset of BBC articles organized by category. This [article](https://www.kaggle.com/yufengdev/bbc-text-categorization) should give you a sense of the level of coding we won’t have to do because we are using Ludwig.

**Comment by SubhradeepSS** [comment]
@TeAmP0is0N I am a beginner in deep learning.I have worked on a text summarization project during JGEC Winter of Code.
I am interested to work on the issue.Will it be ok for a beginner like me to proceed upon the issue?

**Comment by RohitRathore1** [comment]
@SubhradeepSS Yeah, you can proceed with it. The issue is open for all. Go ahead.

**Comment by deep1401** [comment]
Hi, would you be open to using Zero Shot Learning for this task? I could make a notebook to demonstrate on certain datasets you'll suggest using the Huggingface models.
